src/components/snapshot/SnapshotPopup.jsx

```diff
@@ -95,7 +96,19 @@ export class SnapshotPopup extends React.PureComponent {
   componentDidUpdate(prevProps) {
     const { dispatch, snapshot, policies } = this.props;
     if (_.isEmpty(prevProps.snapshot) && !_.isEmpty(snapshot)) {
-      dispatch(getSnapshotById(snapshot.id));
+      dispatch(
+        getSnapshotById({
+          snapshotId: snapshot.id,
+          include: [
+            SNAPSHOT_INCLUDE_OPTIONS.SOURCES,
+            SNAPSHOT_INCLUDE_OPTIONS.TABLES,
+            SNAPSHOT_INCLUDE_OPTIONS.RELATIONSHIPS,
+            SNAPSHOT_INCLUDE_OPTIONS.ACCESS_INFORMATION,
+            SNAPSHOT_INCLUDE_OPTIONS.PROFILE,
+            SNAPSHOT_INCLUDE_OPTIONS.DATA_PROJECT,
+          ],
```

Comment:
Do we already need all of these options?

Reply:
I wanted to have them all there to make sure that we get all of the potential information we might need